src/routes/RemediationDetailsComponents/ExecutionHistoryContent/Columns.js

```diff
@@ -0,0 +1,31 @@
+import { wrappable } from '@patternfly/react-table';
+import {
+  SystemNameCell,
+  InsightsConnectCell,
+  ExecutionStatusCell,
+} from './Cells';
+import { renderComponent } from '../helpers';
+
+export default [
+  {
+    title: 'System name',
+    transforms: [wrappable],
+    // sortable: 'action',
```

Comment:
Do we plan to make them sortable or not? If the answer is no we can delete these commented out pieces

Reply:
We do plan to make this sortable, the backend just hasnt merged this to production. Since not everyone is familiar with ATT I wanted to leave this in just incase this gets done later on. To make sorting work we'd just have to uncomment the lines and change the word to whatever the BE param ends up being